anchor/validator_store/src/lib.rs

```diff
@@ -1246,6 +1280,10 @@ impl<T: SlotClock, E: EthSpec> ValidatorStore for AnchorValidatorStore<T, E> {
                     version: DATA_VERSION_PHASE0,
                     data_ssz: DataSsz::Contributions(data).as_ssz_bytes(),
                 },
+                self.get_instant_in_slot(
+                    slot,
+                    Duration::from_secs(self.spec.seconds_per_slot) * 2 / 3,
+                )?,
```

Comment:
Could we extract this and the similar cases above into variables to improve readability?